gen3/bin/gitops.sh

```diff
@@ -385,13 +387,43 @@ gen3_gitops_sync() {
     portal_roll=true
   fi
 
+  if [[ ! -f "${commonsManifestDir}/etlMapping.yaml" ]]; then
+    gen3_log_info "etl mapping file not found, skipping etl update"
+  else
+    local filename
+    local configMapFile
+    local comparingFile
+    local diffMsg
+    diffMsg=""
+    filename="etlMapping.yaml"
+    commonsFilepath="${commonsManifestDir}/$filename"
+    configMapFile=$(g3kubectl get cm etl-mapping -o json | jq -r '.data."'"$filename"'"' | tr -d '\n' | base64 -w 0)
+    # get file contents from default file or commons's file
```

Comment:
I know the portal-diff stuff does this base64 stuff, but that's because the secret is base64 encoded.
I think you can remove the `base64` call for the etl-diff - configmap is not encoded